teeio-validator/include/ide_test.h

```diff
@@ -340,14 +340,44 @@ struct _ide_common_test_switch_internal_conn_context_t {
   ide_common_test_port_context_t dps;
 };
 
+typedef enum {
+  IDE_COMMON_TEST_CASE_ASSERTION_RESULT_SKIPPED = 0,
+  IDE_COMMON_TEST_CASE_ASSERTION_RESULT_SUCCESS = 1,
+  IDE_COMMON_TEST_CASE_ASSERTION_RESULT_FAILED = 2
+} IDE_COMMON_TEST_CASE_ASSERTION_RESULT;
```

Comment:
type should be: `ide_common_test_case_assertion_result_t`.

Reply:
Updated.